### PR TITLE
make autotest really work

### PR DESCRIPTION
**Pull request by Jackhr-arch**
If you've ever looked at the action log, you'll see "USAGE: python -m hust_pass <option>" instead of success.
More importantly, tesseract is not installed, and you have not set Uid and Pwd in secrets, which makes automatic checking a joke.
Since there seems to be no effective way to install tesseract through cli under windows, the automatic test on windows will always fail, you can find a way by yourself or delete 'windows-latest' in the matrix.os .

In summary, if you want automatic checking to work properly, you need to set Uid and Pwd in secrets.

### CODE REVIEW

**Comment by Jackhr-arch** [comment]
NOTE: Since you have not set Uid and Pwd, CI will always fail, too

**Comment by MarvinTerry** [comment]
I almost forget to add my Uid/Pwd

But this can be risky, and I should look super carefully next time somebody bring up a PR.

And I think simply testing it on Linux is well enough. As a python-lib, compatibility is well handled.

**Comment by MarvinTerry** [comment]
Oh, wait a minute, I check it tomorrow to see if ```python -m hust_login --autotest``` prints out any of my personal info on the console.

Since anyone can have a glance at Actions' console.

**Comment by MarvinTerry** [comment]
BTW, GitHub secrets' name is set to be in uppercase.

Not sure if you should fix ```Uid``` to ```UID```

**Comment by MarvinTerry** [comment]
Since everyone who commits a PR can have a test with my account, that'd be too risky if someone makes a devil PR on purpose.

Unfortunately, I decide not to provide my own Uid/Pwd due to a privacy concern.

**Comment by Jackhr-arch** [comment]
I have done test, and Action does hide the secrets. Here is the [link](https://github.com/jack1130/HustLogin/actions/runs/5805208687/job/15735949862) ( I set them to 'test', it seems that action hides them by regular matching)
If you decide not to provide, I recommand you to disable autotest to avoid daily email from action :-P

**Comment by MarvinTerry** [comment]
Okay, BTW, inspired by your yml file, I set up a private repo pushing daily info to my iPhone via BARK, and I think it can be a way to check out if our lib works.

Here's the code: 

1. yml
```yml
name: 'push_curriculum_freeroom'

on:
  schedule:
    - cron: '50 23 * * *'
  push:

jobs:
  run_script:
    name: Run Python Script
    runs-on: ubuntu-latest

    steps:
      - name: Checkout Repository
        uses: actions/checkout@v3

      - name: Set up Python
        uses: actions/setup-python@v3
        with:
          python-version: 3.11  # Replace 'x' with your desired Python version

      - name: Install hust_login package
        run: pip install hust_login

      - name: Install tesseract-ocr
        run: |
            sudo apt-get update
            sudo apt-get install tesseract-ocr

      - name: Run Python Script
        env:
          BARK_TOKEN: ${{ secrets.BARK_TOKEN }}
          HUST_UID: ${{ secrets.HUST_UID }}
          HUST_PWD: ${{ secrets.HUST_PWD }}
        run: |
          python push_curriculum.py
          python push_freeroom.py
```

2. push_curriculum.py
```python
import requests
from datetime import datetime
import hust_login
import os

Uid = os.environ.get('HUST_UID')
Pwd = os.environ.get('HUST_PWD')
device_key = os.environ.get('BARK_TOKEN')

url = 'https://api.day.app/push'

payload = {
    "device_key": device_key,
    "title": "Curriculum",
    "body": "Test Bark Server",
    "sound": "telegraph",
    "group": "Curriculum"
}

date_today = datetime.today().isoformat()

with hust_login.HustPass(Uid,Pwd) as s:
    data = s.QuerySchedules(date_today)[0]

push_text = ''

push_text += '{} {}\n'.format(data['date'],datetime.fromisoformat(data['date']).strftime('%A'))
for course in data['curriculum']:
    push_text += '第{}节: '.format(course['No'])
    push_text += '{0:{1}<9}\t'.format(course['course'][:9],chr(12288))
    push_text += '{}\n'.format(course['place'])

payload['body'] = push_text

requests.post(url,payload)
```

3. push_freeroom.py
```python
import requests
from datetime import datetime
import hust_login
import os

Uid = os.environ.get('HUST_UID')
Pwd = os.environ.get('HUST_PWD')
device_key = os.environ.get('BARK_TOKEN')

url = 'https://api.day.app/push'

payload = {
    "device_key": device_key,
    "title": "FreeRoom",
    "body": "Test Bark Server",
    "sound": "telegraph",
    "group": "FreeRoom"
}

date_today = datetime.today().isoformat()

with hust_login.HustPass(Uid,Pwd) as s:
    data = s.QueryFreeRooms(date_today)['buildings']['东九楼A']

push_text = ''

push_text += '{} {}\n'.format(date_today,datetime.fromisoformat(date_today).strftime('%A'))
for i in range(0,len(data),2):
    push_text += '第{}~{}节:\n'.format(i+1,i+2)
    roomlist = [room for room in data[i]['roomlist'] if room in data[i+1]['roomlist']]
    for room in roomlist:
        push_text += '{} '.format(room)
    push_text += '\n'
    
payload['body'] = push_text

requests.post(url,payload)
```

**Comment by Jackhr-arch** [comment]
Great idea. In this case, I will trim the tests in CI and del autotest. Another pr is on the way
